package.json

```diff
@@ -2,8 +2,10 @@
   "name": "libp2p-noise",
   "version": "1.1.2",
   "main": "dist/src/index.js",
+  "types": "dist/src/dist/index.d.ts",
```

Comment:
This seems to be a typo, but you really dont need to specify the types field when you have the .d.ts next to the source file